public/html/index.html

```diff
@@ -9,6 +9,34 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link href="/assets/base.css" rel="stylesheet" />
+    <link
```

Comment:
Should we also pre-load the CSS? `<link href="/assets/base.css"`

Reply:
Afaik we don't need to because it's already in the head?

Reply:
<img width="425" alt="image" src="https://github.com/leather-wallet/extension/assets/1618764/a0887276-2f14-4921-8988-83af50f5b30b">